init-tools.sh

```diff
@@ -64,6 +64,9 @@ get_current_linux_name() {
 }
 
 if [ -z "$__DOTNET_PKG" ]; then
+    if [ "$(uname -m | grep "i[3456]86")" = "i686" ]; then
```

Comment:
We expect to have Linux x86 Docker image come up soon. Is the inability to build on x86 a function of that image not being available (but if it were to be, these changes would just work)?